packages/cli-kit/src/git.ts

```diff
@@ -12,12 +13,61 @@ export const GitNotPresentError = () => {
   )
 }
 
+export const OutsideGitDirectoryError = () => {
+  return new Abort('Must be inside a Git directory to continue')
+}
+
+export const NoCommitError = () => {
+  return new Abort(
+    'Must have at least one commit to run command',
+    content`Run ${token.genericShellCommand("git commit -m 'Initial commit'")} to create your first commit.`,
```

Comment:
Should we tell them to `git add` first?

Reply:
I did not do so since git itself will tell you if it has nothing to commit and provide solutions.